bootstrap/app.php

```diff
@@ -99,12 +100,15 @@
 |
 */
 
-//$app->register(App\Providers\AuthServiceProvider::class);
+$app->register(App\Providers\AuthServiceProvider::class);
 $app->register(\Domains\Accounts\AccountsServiceProvider::class);
 $app->register(\Domains\Links\LinksServiceProvider::class);
 $app->register(\Domains\Tags\TagsServiceProvider::class);
 $app->register(\Illuminate\Mail\MailServiceProvider::class);
 $app->register(\Illuminate\Notifications\NotificationServiceProvider::class);
+$app->register(Laravel\Passport\PassportServiceProvider::class);
+$app->register(Dusterio\LumenPassport\PassportServiceProvider::class);
+$app->register(GrahamCampbell\Throttle\ThrottleServiceProvider::class);
```

Comment:
Let's add the leading `\` for consistency and keep the entries alphabetically ordered for easier future reading.

Reply:
Thanks, changed!